test/data/manifests/fedora_38-aarch64-iot_raw_image-boot.json

```diff
@@ -2207,6 +2207,13 @@
           {
             "type": "org.osbuild.copy",
             "inputs": {
+              "ostree-tree": {
+                "type": "org.osbuild.ostree.checkout",
+                "origin": "org.osbuild.source",
+                "references": [
+                  ""
```

Comment:
Shouldn't there be some reference? An empty string does not look correct.

Reply:
This is empty because there is no commit checksum available.

Reply:
Right. As we discussed, it would be good to use some placeholder commit for test manifests, because this looked suspicious at the first sight. Thanks for the explanation.